tapes/submitters.tape

```diff
@@ -2,7 +2,6 @@
 # TASK, REALIZATION, CONFIGURATION: variables passed by ducttape
 submitter shell :: COMMANDS TASK_VARIABLES {
   action run {
-    set +u
```

Comment:
Getting rid of this still crashes for me. I don't think I quite understand what you are trying to do here.

Reply:
How does it crash? You don’t need `set +u`: bash can instead use default values with this syntax:

    echo ${var:-}

where what’s after the “-“ (here, the empty string) is the default value if $var is unset.

Reply:
I got this upon `source virtualenv` is being called:

```
_OLD_VIRTUAL_PATH: unbound variable
```

I digged around a bit and remembered that it was a [virtualenv bug](https://github.com/pypa/virtualenv/pull/922) that was fixed 7 hours ago (!). Maybe I should update my virtualenv some day, but can I revert it for this test?

Reply:
Actually maybe let's keep the `set +u` around for a while, because after python 3.6 virtualenv is released with python -- it may take a while for everyone to get up-to-date with this fix.